examples/react-app/src/App.test.tsx

```diff
@@ -8,6 +8,8 @@ import { trackMessage, AnalyticsEvent } from "@amplitude/analytics-messages";
 const SELECTOR_EXPERIMENT_EXPOSURE_BUTTON = /experiment.exposure/i
 const SELECTOR_ANALYTICS_USER_LOGGED_IN_BUTTON = /login \(with event method/i
 const SELECTOR_ANALYTICS_USER_SIGNED_UP_BUTTON = /sign up/i
+const SELECTOR_USER_SET_USER_ID_BUTTON = /Set User Id/i
```

Comment:
Minor, do we need this? Seems not being used anywhere.

Reply:
Good catch. Fixed.